examples/map/execute_on_key/main.go

```diff
@@ -20,30 +20,41 @@ func main() {
 	if err != nil {
 		log.Fatal(err)
 	}
-	// Populate map
+	// Populate map.
 	if _, err = m.Put(ctx, "key1", "value"); err != nil {
 		log.Fatal(err)
 	}
 	if _, err = m.Put(ctx, "key2", "value"); err != nil {
 		log.Fatal(err)
 	}
-	// only change value corresponding to key1
-	finalVal, err := m.ExecuteOnKey(ctx, &IdentifiedEntryProcessor{value: "test"}, "key1")
+	// Only change value corresponding to "key1".
+	prevVal, err := m.ExecuteOnKey(ctx, &IdentifiedEntryProcessor{value: "testOnKey"}, "key1")
 	if err != nil {
 		log.Fatal(err)
 	}
-	fmt.Println("previous value:", finalVal)
-	// check the values
-	value, err := m.Get(ctx, "key1")
+	fmt.Println("previous value:", prevVal)
+	// Observe value corresponding to "key1" changed, "key2" remains unchanged.
+	checkValue(ctx, m, "key1")
+	checkValue(ctx, m, "key2")
+	// Change both entries.
+	updatedTo, err := m.ExecuteOnKeys(ctx, &IdentifiedEntryProcessor{value: "testOnKeys"}, "key1", "key2")
 	if err != nil {
 		log.Fatal(err)
 	}
-	fmt.Println("key1:", value)
-	value, err = m.Get(ctx, "key2")
+	// ExecuteOnKeys returns updated results.
+	fmt.Println(updatedTo)
+	// check the values
+	checkValue(ctx, m, "key1")
+	checkValue(ctx, m, "key2")
+	// Shutdown client
+	client.Shutdown(ctx)
+}
+
+// Retrieve and print value corresponding to "key" from map "m"
+func checkValue(ctx context.Context, m *hazelcast.Map, key string) {
```

Comment:
No need to use context for this example, let's just use `context.TODO()` to make it simpler.

Reply:
Fixed at https://github.com/hazelcast/hazelcast-go-client/pull/720/commits/014319c28f64f58430797e915b99c3bf348410dc